modules/installing-ocp-agent-gather-log.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-gather-log_{context}"]
+= Gathering log data from a failed Agent-based installation
+
+Use the following procedure to gather data about a failed Agent-based installation to provide for a support case.
+
+.Procedure
+
+. Run the following command and collect its output:
++
+[source,terminal]
+----
+$ ./openshift-install --dir <install_directory> agent wait-for bootstrap-complete --log-level=debug
+----
+
+. If the output from the previous command indicates a failure, or if the bootstrap is not progressing, run the following command for each node and collect its output:
++
+[source,terminal]
+----
+$ ssh core@<node-ip> sudo /usr/local/bin/agent-gather -O > <local_tmp_path>/agent-gather.tar.xz
+----
++
+[NOTE]
+====
+It is helpful to gather data from each node, although it is only necessary to gather data from node 0.
+====
+
+. If the bootstrap completes and the cluster nodes reboot, run the following command and collect its output:
++
+[source,terminal]
+----
+$ ./openshift-install --dir <install_directory> agent wait-for install-complete --log-level=debug
+----
+
+. If the output from the previous command indicates a failure, perform the following steps:
+
+.. Run the following command:
++
+[source,terminal]
+----
+$ oc adm must-gather
+----
+//to-do: check whether the must gather command needs any specific arguments in order to gather data specific to the agent install. The KCS this procedure is based on says to "Use the kubeconfig file available on <installation directory>/auth." but I don't know what that means in this case.
```

Comment:
No special arguments would be needed, but if an agent install does not save the kubeconfig to this same path you may want to describe where to grab the kubeconfig.

Reply:
Yes the agent installer does save the kubeconfig to this path.

Reply:
Just confirmed with Bob, it is saved to the same path. Thanks!